string my_color
int32 my_points
int32 enemys_points
int32[18] point_state
